### PR TITLE
Fix accessibility issues

### PR DESCRIPTION
**Pull request by teckim**
## Accessibility report:

- Page titles (no issues found)
- Image text alternatives** (`alt text`) 5a30bb0cd1cb1a020200982398676fc24ff8dc99
- Text headings (no issues found)
- Color contrast 143605f459aac3d3e7a7a1e5f815a1fa0fef2914
- Resize (no issues found)
- Interaction (no issues found)
- Moving content (no issues found)
- Multimedia (no issues found)
- The basic structure of the page (no issues found)

### CODE REVIEW

File: src/css/components/contact.css [hunk 1]
```diff
@@ -24,7 +24,7 @@
 .contact .contact__headline-text {
   font-size: var(--text-lg);
   line-height: 28px;
-  color: var(--color-primary-B50);
+  color: var(--color-neutrals);
 }
 
 .contact .contact__form {
```
Comment (OLIPLICHE) [inline src/css/components/contact.css]: - [ ] Please work on the color contrast of the `button Get In Touch`  I suggest that you re-adust the border color when Hover. 
- See Details
c2

- [ ] More Details:
c

File: src/index.html [hunk 2]
```diff
@@ -64,27 +64,27 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
         <ul class="social-stack">
```
Comment (OLIPLICHE) [inline src/index.html:64]: - [ ]
l
 
- [ ] Please make sure It's possible to navigate through the page(Portfolio, About, Contact), focus on its elements.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is descriptive 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 2 above)
Comment (kelomo2502) [reply src/index.html:64]: Dear reviewer, thank you for doing a good job. However, I am obliged to know if this particular change was part of the project requirements. Thank you

File: src/index.html [hunk 3]
```diff
@@ -64,33 +64,33 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
...
             <a>
-              <img src="./assets/icons/medium.svg" alt="" />
+              <img src="./assets/icons/medium.svg" alt="medium icon" />
             </a>
           </li>
           <li>
             <a>
-              <img src="./assets/icons/github.svg" alt="" />
+              <img src="./assets/icons/github.svg" alt="github icon" />
             </a>
```
Comment (aimemalaika) [inline src/index.html:83]: - [x] [OPTIONAL] Kindly add links foreach `a` tags

**Review by aimemalaika — APPROVED** [review]
Hi @teckim & @kelomo2502, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 3 above)
Comment (teckim) [reply src/index.html:83]: thank you for your seggestion ^^

**Pull request merged**